apis/traffic-split/v1alpha4/traffic-split.md

```diff
@@ -28,6 +28,9 @@ configuration would continue to operate.
 Implementations will weight outgoing traffic between the services referenced by
 `spec.backends`. Each backend is a Kubernetes service that potentially has a
 different selector and type. Weights must be whole numbers.
+Validation beyond what is in the
+[TrafficSplit OpenAPI schema](https://github.com/servicemeshinterface/smi-sdk-go/blob/main/crds/split.yaml),
+such as checking for negative weights, is left up to the implementation.
```

Comment:
I think the "such as checking for negative weights" phrase makes this line a little confusing.

Reply:
Good point. I'll remove it.